servers/zms/src/main/java/com/yahoo/athenz/zms/notification/RoleMemberReviewNotificationTask.java

```diff
@@ -0,0 +1,159 @@
+/*
+ *  Copyright 2020 Verizon Media
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.yahoo.athenz.zms.notification;
+
+import com.yahoo.athenz.common.server.notification.*;
+import com.yahoo.athenz.zms.DBService;
+import com.yahoo.athenz.zms.DomainRoleMember;
+import com.yahoo.athenz.zms.MemberRole;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import static com.yahoo.athenz.common.server.notification.NotificationServiceConstants.*;
+
+public class RoleMemberReviewNotificationTask implements NotificationTask {
+    private final DBService dbService;
+    private final RoleMemberNotificationCommon roleMemberNotificationCommon;
+    private static final Logger LOGGER = LoggerFactory.getLogger(RoleMemberReviewNotificationTask.class);
+    private final static String DESCRIPTION = "Review before expiration reminder";
```

Comment:
A better description would be: "Periodic Review Reminder"

this feature is not really about expiration. it's just about reviewing members without expiring anyone.